cdk/package.json

```diff
@@ -25,8 +25,10 @@
   },
   "dependencies": {
     "@aws-sdk/client-ec2": "^3.749.0",
-    "aws-cdk-lib": "^2.189.1",
+    "aws-cdk-lib": "^2.187.0",
```

Comment:
we downgraded aws-cdk-lib to avoid the peer dependency issue (c.f. https://github.com/aws-samples/cdk-image-pipeline/issues/258)